readme.md

```diff
@@ -38,63 +38,63 @@ $ npm -v
 
 * **Question:** What is the difference between forking and cloning a repository as opposed to just cloning a repository?
 
-* **Your Answer:**
+* **Your Answer: Forking allows the user to make a copy of a repository on their local Github branch. Cloning allows for you to copy a remote Git repo onto a local machine**
```

Comment:
To be more precise, it allows you to make a copy of the repository on your GitHub. It doesn't necessarily have anything to do with a branch.